src/drivers/indexeddb.js

```diff
@@ -70,7 +70,8 @@ function _checkBlobSupport(idb) {
     if (typeof supportsBlobs === 'boolean') {
         return Promise.resolve(supportsBlobs);
     }
-    return _checkBlobSupportWithoutCaching(idb).then(function(value) {
+    var txn = idb.transaction(DETECT_BLOB_SUPPORT_STORE, 'readwrite');
```

Comment:
Can you expand this variable name? I assume it means `transaction`, but I like verbose variable names 😄